README.md

```diff
@@ -35,6 +35,7 @@ Shared service to generate PNG/PDF snapshots of our websites.
 - `logo` — (optional) Display your site's logo in the header area of each page on your PDF. See [Custom Logos](#custom-logos) section for instructions on adding your logo to this repository.
 - `user` — (optional) HTTP Basic Authentication username.
 - `pass` — (optional) HTTP Basic Authentication password.
+- `cookies` — (optional) a String representing browser cookies. Just send the contents of `document.cookie` and it should work.
```

Comment:
"just" and "should"... any debugging techniques here?

Suggest "Sending the contents of `document.cookie` will typically work." to me this suggests that you can send more than one cookie. Any extra details that someone might want to know or is the short hand of referencing `document.cookie` sufficient?

Reply:
The (lack of) pluralization is legacy. As a FED, when I "set a cookie" i'm concerned with setting a single key/value pair in the cookie object. Other services might set their own cookie. A long time ago the browsers were calling this `document.cookie` and because it would have caused major compatibility issues to rename, it was never renamed. But you can put many key/val pairs in there, so `document.cookie` holds multiple cookies. This [MDN article](https://developer.mozilla.org/en-US/docs/Web/API/Document/cookie) refers to multiple cookies inside `document.cookie` in the same way.

Setting a cookie means it gets appended to `document.cookie` which is accessible by opening a JS console, typing `document.cookie` and hitting enter. You'll get a list of all key/val pairs. Here's my SitRep cookies as an example with some Xs thrown in:

```
"_ga=GA1.2.186XXXXX99.15XXXXXXXX; __unam=25ebe74-165a01XXXXX-6cXXXXXX-6; _gid=GA1.2.180XXXXX44.15XXXXXXXX; _gat_UA-1234XXXXX-X=1; locale=en"
```

Of course, the `locale=en` is the important part for SitRep, but sending the GA stuff — or perhaps a PHP session cookie if a future requesting service had logged in users generating Snaps — helps render the webpage as the specific visitor sees it.

For that reason, I'm recommending to requesting services that they send the entire contents of `document.cookie` to achieve WYSIWYG without too much fuss on their end.